elliottlib/errata.py

```diff
@@ -125,6 +125,22 @@ def remove_multi_jira_issues(advisory_id, jira_list: List):
     return [remove_jira_issue(advisory_id, jira_id) for jira_id in jira_list]
 
 
+def remove_bug(advisory_id, bug_id):
+    """
+    Remove a bug from advisory
+    Response code will return
+    """
+    return ErrataConnector()._post(f"/api/v1/erratum/{advisory_id}/remove_bug", data={"bug": f"{bug_id}"})
+
+
+def remove_multi_bugs(advisory_id, bug_list: List):
+    """
+    Remove multi bugs from advisory
+    Return a list of response code
+    """
+    return [remove_bug(advisory_id, bug_id) for bug_id in bug_list]
```

Comment:
This is not performant. For N bugs, it will create N `ErrataConnector` instances. Each will create a new TCP connection and kerberos authentication.

Reply:
yes, but errata don't have batch remove / add api, same for jira issues

Reply:
I think what yuxiang is suggesting is to use the same ErrataConnector instance for each request